src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -431,7 +431,12 @@ public async Task RemoveAccountAsync(ApplicationConfiguration appConfig, IAccoun
                         }
                         else
                         {
-                            WamAdapters.ThrowExceptionFromWamError(result, _logger);
+                            _logger?.WarningPii(
+                            $"[WamBroker] Could not sign out user {account.Username} - error: {result.Error}",
+                            $"[WamBroker] Could not sign out user, error: {result.Error}");
+
+                            string errorMessage = $"{result.Error} (error code : {result.Error.ErrorCode})";
+                            throw new MsalServiceException("wam_sign_out_failed", errorMessage);
```

Comment:
IMO, the SingOut operation is best effort. We (MSAL.NET) will delete all artefacts from our side.

And then we call broker to sign out the user as well. There seem to be cases where this won't work, because we don't have the correct account id (we use home_oid, MSAL C++ uses local_oid). 

It might be better to do `try { singout from broker} catch { log exception as error, but continue }`